pkg/utils/system/system.go

```diff
@@ -66,22 +62,13 @@ func GetCapabilityDir() (string, error) {
 	return filepath.Join(home, "capabilities"), nil
 }
 
-// GetEnvDir return KubeVela environments dir
-func GetEnvDir() (string, error) {
-	homedir, err := GetVelaHomeDir()
-	if err != nil {
-		return "", err
-	}
-	return filepath.Join(homedir, "envs"), nil
-}
-
-// GetCurrentEnvPath return current env config
-func GetCurrentEnvPath() (string, error) {
+// GetEnvDirByName will get env dir from name
+func GetEnvDirByName(name string) string {
```

Comment:
why do we still need a env dir when ENV is a CRD?

Reply:
The `vela configs` command relies on `envName` as the local index directory.